ui/src/version/deployment/components/EndpointDeployment.js

```diff
@@ -24,15 +24,18 @@ import {
   EuiFlexItem,
```

Comment:
Can you provide gif and/or screenshot for the UI changes?

Reply:
Added to the PR description